### PR TITLE
Refactoring updates

### PR DESCRIPTION
**Pull request by YahorKazlou**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +42,24 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-react": "^7.37.1",
     "globals": "^15.11.0",
-    "prettier": "3.3.3"
+    "husky": "^9.1.6",
+    "lint-staged": "^15.2.10",
+    "prettier": "3.3.3",
+    "prettierr": "^1.15.3-dev"
+  },
+  "prettier": {
```
Comment (Den4ik5) [inline package.json]: please move to prettierrc

File: package.json [hunk 2]
```diff
@@ -41,6 +42,24 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-react": "^7.37.1",
     "globals": "^15.11.0",
-    "prettier": "3.3.3"
+    "husky": "^9.1.6",
+    "lint-staged": "^15.2.10",
+    "prettier": "3.3.3",
+    "prettierr": "^1.15.3-dev"
```
Comment (Den4ik5) [inline package.json]: what is this used for

File: package.json [hunk 3]
```diff
@@ -41,6 +42,24 @@
...
+    "printWidth": 80,
+    "semi": true,
+    "tabWidth": 4
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
```
Comment (Den4ik5) [inline package.json]: please check package.json. structure 
you should have scripts section 
add script for prettier

File: src/components/Navigation/DesktopNavigation/DropdownElement.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+const cx = classNames.bind(styles);
+
+const DropdownElement = ({ type, name, icon }) => (
+    <li>
+        {type === NAV_ELEMENT_TYPES.SUBHEADER ? (
+            <div className={styles.navbarItem}>{name}</div>
+        ) : (
+            <a
+                className={cx({
+                    navbarItem: true,
```
Comment (Den4ik5) [inline src/components/Navigation/DesktopNavigation/DropdownElement.jsx:14]: navbarItem: true, are you sure we can't replace it with navbarItem only

File: package.json (hunk 1 above)
Comment (YahorKazlou) [reply package.json]: Done

File: package.json (hunk 2 above)
Comment (YahorKazlou) [reply package.json]: Husky allows you to add git hooks. Here, `pre-commit` is configured to run `lint-staged` every time I try to commit.
Lint-staged works together with Husky and runs linter and formatter only on changed files. Here, it runs `prettier` on all files with the `.jsx` extension.
Prettierr is a bug. Removed.
Done.

File: package.json (hunk 3 above)
Comment (YahorKazlou) [reply package.json]: Done

File: src/components/Navigation/DesktopNavigation/DropdownElement.jsx (hunk 4 above)
Comment (YahorKazlou) [reply src/components/Navigation/DesktopNavigation/DropdownElement.jsx:14]: We cannot use shorter notation for navbarItem property as we haven't defined it as separate variable anywhere else

**Pull request merged**